UCPKN-2790: Drop D9.4 support and use PHP 8.1/8.2.

kp77 opened the pull request.

kp77 committed:
UCPKN-2790: Drop D9.4 support and use PHP 8.1/8.2.

brummbar commented:
Can you update also the info files to remove the 9.4? We are also dropping Drush 11 in favor of 12. Oe_content you can put the stable release now. Rest looks good.

kp77 committed:
UCPKN-2790: Remove core 9.4 support from info files.

kp77 committed:
UCPKN-2790: Upgrade Drush to version 12.

kp77 committed:
UCPKN-2790: Update drone matrix, remove core 10.0, add 10.2.

kp77 committed:
UCPKN-2790: Upgrade oe_content to 3.1 stable.

kp77 committed:
UCPKN-2790: Fix phpcs, remove spaces around equals for strict_types.

kp77 committed:
UCPKN-2790: Switch to Selenium 4.

kp77 commented:
> Can you update also the info files to remove the 9.4? We are also dropping Drush 11 in favor of 12. Oe_content you can put the stable release now. Rest looks good.

Hi @brummbar,
- removed 9.4 from info files
- upgraded Drush to 12
- upgraded OE Content
- also, aligned with other OE projects: updated the drone test matrix (removed 10.0 and added 10.2) and switched to selenium 4
